portfolio/src/main/webapp/script.js

```diff
@@ -252,24 +254,53 @@ function createListElement(comment) {
   return listElem;
 }
 
+/**
+ * Formats upvote and downvote buttons for each comment. Sets
+ * each button to pressed or unpressed based on data about
+ * whether the current user has pressed either received
+ * from server. 
+ */
 function formatCommentVoteButtons(comment, thisCommentDiv) {
   const upvoteText = document.createElement("p");
   upvoteText.className = "vote-buttons";
   upvoteText.id = `${comment["id"]}-up`;
   upvoteText.innerText = comment["upvotes"];
+  const whichPressed = comment["currUserStatus"];
```

Comment:
add a comment explaining the possible states of this variable and what each means.